dev/infra/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update && apt-get install -y \
     pkg-config  \
     curl  \
     wget  \
-    openjdk-8-jdk  \
+    openjdk-17-jdk-headless  \
```

Comment:
@dongjoon-hyun @HyukjinKwon should we change this?

Reply:
cc @Yikun

Reply:
@dongjoon-hyun So in this PR, should we revert this change and consider it as a followup?

Reply:
Emm, jdk install in here seems useless, because we are using github action to install the java. But if CI passed, it is ok.

Reply:
Thanks @Yikun ~ Let me continue to monitor the running status of GA.